File: PRAK401-2310817210021-RAKHMAD FAUZHAN RAMADHAN.c
#include <stdio.h>

void urutan(int kelipatan, char simbol) {
for (int i = 1; i <= 50; i++) {
if (i % kelipatan == 0) {
printf("%c ", simbol);
} else {
printf("%d ", i);
}
}
}

int main() {
int kelipatan;
char simbol;

scanf("%d %c", &kelipatan, &simbol);

urutan(kelipatan, simbol);

return 0;
}

File: PRAK401-2310817210021-RAKHMAD FAUZHAN RAMADHAN.py
def main():
    input_str = input().split()
    kelipatan = int(input_str[0])
    simbol = input_str[1]

    for i in range(1, 51):
        if i % kelipatan == 0:
            print(simbol, end=" ")
        else:
            print(i, end=" ")
    
if __name__ == "__main__":
    main()
